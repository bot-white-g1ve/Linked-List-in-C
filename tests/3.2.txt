NEW 1
{0}
NEW 1
a
INSERT 0 0 {0}
VIEW 0
VIEW-NESTED 0
NEW 1
{0}
NEW 1
{1}
REMOVE 0
NEW 1
b
INSERT 0 0 {2}