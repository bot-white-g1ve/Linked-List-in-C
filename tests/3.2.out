INVALID COMMAND: NEW
List 0: a
INVALID COMMAND: INSERT
a
a
Nested 1: {List 0}
INVALID COMMAND: NEW
INVALID COMMAND: REMOVE
List 2: b
INVALID COMMAND: INSERT
